sleap/nn/tracker/components.py

```diff
@@ -71,7 +71,8 @@ def factory_object_keypoint_similarity(
 
     """
     keypoint_errors = 1 if keypoint_errors is None else keypoint_errors
-    kp_precision = 1 / (2 * np.array(keypoint_errors) ** 2)
+    with np.errstate(divide='ignore'):
```

Comment:
If you want to ignore key points for the tracking, you can set the error to 0 for the specific key point.
This commit silences the "divide by zero" warnings, as this is an allowed operation.